.konchrc

```diff
@@ -1,22 +1,19 @@
+#!/opt/conda/bin/python
+# -*- coding: utf-8 -*-
 # vi: set ft=python :
 from pathlib import Path
 
 import konch
+import sys
+import os
 from test_konch import env
 import IPython
 import bpython
 import ptpython
 
 konch.config(
     {
-        "context": [
-            konch,
-            env,
-            Path,
-            IPython,
-            bpython,
-            ptpython,
-        ],
+        "context": [sys, os, konch, env, Path, IPython, bpython, ptpython],
```

Comment:
This change (and the associated imports) is unrelated the fix, so please don't include them in the PR.